pkgs/development/libraries/ncurses/default.nix

```diff
@@ -68,6 +77,7 @@ stdenv.mkDerivation rec {
       "--bindir=$dev/bin"
       "--mandir=$man/share/man"
       "--with-pkg-config-libdir=$PKG_CONFIG_LIBDIR"
+      ${lib.optionalString symversion "--with-versioned-symbols=$PWD/package/ncursestw.map"}
```

Comment:
Honestly I'm not 100% sure what the right approach here is.

I was hoping this would address some issues I had been running into around "no version information available" noise but it doesn't seem to be the case (at least based on a quick local test).